dotnetv3/dynamodb/FromSQL/UpdateItemDataModelExample/UpdateItemDataModelExample/UpdateItemDataModel.cs

```diff
@@ -1,102 +1,40 @@
-using Amazon;
-using Amazon.DynamoDBv2;
-using Amazon.DynamoDBv2.DataModel;
-
-namespace UpdateItemDataModel
+namespace UpdateItemDataModelExample
 {
-    // If you change the table name elsewhere
-    // (as in an app.config in another project),
-    // you'll have to change it here and rebuild.
-    [DynamoDBTable("CustomersOrdersProducts")]
-    public class Entry
-    {
-        [DynamoDBHashKey] // The partition key for the table.
-        public string Id
-        {
-            get; set;
-        }
-        [DynamoDBRangeKey] // The sort key for the table.
-        public string Area
-        {
-            get; set;
-        }
-        [DynamoDBProperty]
-        public int OrderId
-        {
-            get; set;
-        }
-        [DynamoDBProperty]
-        public int OrderCustomer
-        {
-            get; set;
-        }
-        [DynamoDBProperty]
-        public int OrderProduct
-        {
-            get; set;
-        }
-        [DynamoDBProperty]
-        public long OrderDate
-        {
-            get; set;
-        }
-        [DynamoDBProperty]
-        public string OrderStatus
-        {
-            get; set;
-        }
-    }
-
+    using System;
+    using System.Configuration;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.DynamoDBv2;
+    using Amazon.DynamoDBv2.DataModel;
+
+    // snippet-start:[dynamodb.dotnet35.UpdateItemDataModel]
+
+    /// <summary>
+    /// Update the model of items in an Amazon DynamoDB table. The
+    /// example was created using the AWS SDK for .NET version 3.7
+    /// and .NET Core 5.0.
+    /// </summary>
     public class UpdateItemDataModel
     {
-        public static async Task<Entry> UpdateTableItemAsync(IDynamoDBContext context, string id, string status)
-        {
-            // Retrieve the existing order.
-            Entry orderRetrieved = await context.LoadAsync<Entry>(id, "Order");
-
-            // Trap any nulls.
-            if (null == orderRetrieved)
-            {
-                throw new ArgumentException("The ID " + id + " did not identify any current order");
-            }
-
-            // Make sure it's an order.
-            if (orderRetrieved.Area != "Order")
-            {
-                throw new ArgumentException("The ID " + id + " did NOT identify an order, but instead identified a " + orderRetrieved.Area);
-            }
-
-            // Update the status of the order.
-            orderRetrieved.OrderStatus = status;
-            await context.SaveAsync(orderRetrieved);
-
-            // Retrieve the updated item.
-            Entry updatedOrder = await context.LoadAsync<Entry>(id, "Order", new DynamoDBOperationConfig
-            {
-                ConsistentRead = true,
-            });
-
-            return updatedOrder;
-        }
-
-        static void Main(string[] args)
+        /// <summary>
+        /// Retrieves the configuration settings, parses the command line, and
+        /// then UpdateItemDataModel.
```

Comment:
and then calls the UpdateItemDataModel method. (?)

Reply:
fixed